acceptancetests/assess_cross_model_relations.py

```diff
@@ -205,6 +270,19 @@ def offer_endpoint(client, app_name, offer_name=None):
     return offer_url, offer_name
 
 
+def register_user_on_controller(client, username, token):
```

Comment:
Why does register_user fail to address this needed, and could it instead be modified to do so?